web/pdf_page_view.js

```diff
@@ -413,10 +413,11 @@ class PDFPageView {
     const tree = await (!this.structTreeLayer.renderingDone
       ? this.pdfPage.getStructTree()
       : null);
-    const treeDom = this.structTreeLayer?.render(tree);
```

Comment:
Why did you change this, since that could theoretically cause errors!?

Please consider the case where the page is destroyed, i.e. from going out of view, in which case the `structTreeLayer` could be removed by the time the API method `getStructTree` resolves.